be/src/olap/rowset/segment_v2/ordinal_page_index.h

```diff
@@ -55,9 +56,11 @@ class OrdinalPageIndexBuilder {
         _num_pages++;
     }
 
-    Slice finish() {
-        // encoded number of elements
+    Slice finish(rowid_t row_num) {
+        // encoded number of pages
         encode_fixed32_le((uint8_t*)_buffer.data(), _num_pages);
+        // encoded number of rows
+        encode_fixed32_le((uint8_t*)_buffer.data() + 4, row_num);
```

Comment:
I don't think we need to store total number of rows in each column's ordinal index, just use `SegmentFooterPB.num_rows`

Reply:
OK, I will use SegmentFooterPB.num_rows and pass it through constructor